### ISSUE TITLE
depth-anaything-tensorrt-simplified.exe - System Error

### ISSUE THREAD

**Issue by Ashoka74** [spacewalk01/depth-anything-tensorrt#9]
Hi, thank you for this amazing project!

I faced an issue while running the .exe :

`The code execution cannot proceed because opencv_world490d.dll was not found. Reinstalling the program may fix the problem`

The path I added in the Cmaketext.txt for OpenCV is :
PATH\opencv\build\x64\vc16\lib

**EDIT:
Find a workaround by adding the missing file directly in the folder with the .exe and it worked perfectly!**

**Comment by spacewalk01** [comment]
Hi thank you. The path in the Cmaketext.txt for OpenCV should be :
PATH\opencv\build And yes you need to add opencv dll files to the folder where the exe file is

**Comment by Ashoka74** [comment]
Thank you for your quick answer.

The program worked for single images; however, for folders and videos, I got this error : 

`CUDA lazy loading is not enabled. Enabling it can significantly reduce device memory usage and speed up TensorRT initialization. See "Lazy Loading" section of CUDA documentation https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#lazy-loading`

Followed by :

`[ INFO:0@0.845] global videoio_registry.cpp:244 cv::anonymous-namespace'::VideoBackendRegistry::VideoBackendRegistry VIDEOIO: Enabled backends(9, sorted by priority): FFMPEG(1000); GSTREAMER(990); INTEL_MFX(980); MSMF(970); DSHOW(960); CV_IMAGES(950); CV_MJPEG(940); UEYE(930); OBSENSOR(920)
[ INFO:0@0.845] global backend_plugin.cpp:383 cv::impl::getPluginCandidates Found 3 plugin(s) for FFMPEG
[ INFO:0@0.846] global plugin_loader.impl.hpp:67 cv::plugin::impl::DynamicLib::libraryLoad load PATH\Depth-Anything\depth-anything-tensorrt\opencv_videoio_ffmpeg490_64d.dll => FAILED
etc..`

The result is that I am not able to open the output_video file.
I reinstalled GTK manually on Windows but the same error persists.
